src/Screens/PomodoroPage/pomoPage.css

```diff
@@ -0,0 +1,115 @@
+.pomo_body{
+    background: linear-gradient(to right, #304352, #d7d2cc);
+    height: 100vh;
+    width: 100%;
+}
+
+.pomo_container{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: min-content;
+    justify-content: center;
+    align-items: center;
+
+}
+.pomo_button_container{
+    display: flex;
+    width: 28%;
+    height: 4.2rem;
+    justify-content: space-around;
+    align-items: center;
+   background: linear-gradient(to right, #16222a, #3a6073); 
+    padding: 10px;
+    margin-top: 10px;
+    border-radius: 10px;
+}
+.pommo_button_gropus{
+    width: 12rem;
+    height: 4rem;
+    background: linear-gradient(to right, #dc2424, #4a569d);
+    border-radius: 2rem;
+    font-size: 1.8rem;
+    font-weight: 600;
+
+}
+.pomo_watch_container{
+    display: flex;
+    width: 28%;
+    height: 40rem;
+    justify-content: center;
+    align-items: center;
+    background: linear-gradient(to right, #16222a, #3a6073);
+    object-fit: cover;
+    border-image-width: 20px;
+    padding: 10px;
+    margin-top: 10px;
+    border-radius: 10px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
```

Comment:
https://github.com/SumitSaundadkar/mypomodoroapp/pull/1/files#:~:text=.pomo_watch_container%7B-,display%3A%20flex%3B,align%2Ditems%3A%20center%3B,-background%3A%20linear


This already makes the container center. So, there is no use of the above lines

Reply:
done